### PR TITLE
PCC-105 Add contact to prefix

### PR DESCRIPTION
**Pull request by cthermolia-grnet**
Add contact name and email to prefix

### CODE REVIEW

File: api/src/main/resources/db/migration/V1.3__prefix.sql
Comment (ctriant) [inline api/src/main/resources/db/migration/V1.3__prefix.sql:1]: I think it's a better practice to create a new revision V1.4. Let's ask @fbasios

Comment (cthermolia-grnet) [reply api/src/main/resources/db/migration/V1.3__prefix.sql:1]: yes i need to create an update script to prefix table to introduce the new field. until fixed , please dont merge

File: api/src/main/java/gr/grnet/pccapi/endpoint/PrefixEndpoint.java
Comment (fbasios) [inline api/src/main/java/gr/grnet/pccapi/endpoint/PrefixEndpoint.java]: I think we should correct this summary.

**Review by fbasios — APPROVED** [review]

File: api/src/main/java/gr/grnet/pccapi/endpoint/PrefixEndpoint.java
Comment (cthermolia-grnet) [reply api/src/main/java/gr/grnet/pccapi/endpoint/PrefixEndpoint.java]: fixed

**Review by fbasios — APPROVED** [review]

**Pull request merged**
